CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+  * **Required**<br/>
+    This governance action cannot be ratified without achieving the AVST threshold.
+  * **SPO**<br/>
+    For this governance action, SPO vote endorsement is not required if there is a sufficient active voting stake.
+    Otherwise, SPO vote endorsement is required.
+
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Action | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---         | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | Required    | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO         | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | Required    | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO         | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -           | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO         | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO         | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO         | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO         | $R_{6c}$ |
+
+Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
+
+> **Note**
+> For all treasury withdrawals, the withdrawal threshold is the **total** amount of Lovelace that is withdrawn by the action, not the amount of any single withdrawal if the action specifies more than one withdrawal.
+
+##### Governance Action Lifecycle
+
+Governance actions are checked for ratification only on an epoch boundary. This delay allows everyone to vote on each proposal and prove that they are active.
+
+**At most one** governance action **of each type** (i.e. one from each of the six categories listed above) can be staged for enactment in any given epoch. For example, there can only be one treasury withdrawal action in a single epoch (it may, however, comprise many individual withdrawals).
```

Comment:
When we say withdrawal, are we talking about a single action that disperses funds to multiple parties or is it literally just one withdrawal to one party.

Reply:
We are proposing this blanket policy because the constitutional committee has a very important role in vetoing bad actions. Removing lovelace from the treasury should never been taken without much care! The current design already includes bundles, it is mentioned elsewhere in this document.

Reply:
> When we say withdrawal, are we talking about a single action that disperses funds to multiple parties or is it literally just one withdrawal to one party?

it is a map from stake credentials to lovelace

Reply:
> because the constitutional committee has a very important role in vetoing bad actions. Removing lovelace from the treasury should never been taken without much care!

This is what I'm thinking too. Wouldn't it be more transparent to make many more, specific, withdrawls. What happens if a subset of a bundled withdrawl is vetoed? Will it stop the entire withdrawl?

> The current design already includes bundles, it is mentioned elsewhere in this document.

Ok, this makes more sense now that I know this.

Reply:
> What happens if a subset of a bundled withdrawl is vetoed? Will it stop the entire withdrawl?

that's correct.